mob.go

```diff
@@ -45,28 +45,31 @@ func doneSquash(value string) string {
-	GitHooksEnabled                bool   // override with MOB_GIT_HOOKS_ENABLED
-	RequireCommitMessage           bool   // override with MOB_REQUIRE_COMMIT_MESSAGE
-	VoiceCommand                   string // override with MOB_VOICE_COMMAND
-	VoiceMessage                   string // override with MOB_VOICE_MESSAGE
-	NotifyCommand                  string // override with MOB_NOTIFY_COMMAND
-	NotifyMessage                  string // override with MOB_NOTIFY_MESSAGE
-	NextStay                       bool   // override with MOB_NEXT_STAY
-	StartIncludeUncommittedChanges bool   // override with MOB_START_INCLUDE_UNCOMMITTED_CHANGES variable
-	StashName                      string // override with MOB_STASH_NAME
-	WipBranchQualifier             string // override with MOB_WIP_BRANCH_QUALIFIER
-	WipBranchQualifierSeparator    string // override with MOB_WIP_BRANCH_QUALIFIER_SEPARATOR
-	WipBranchPrefix                string // override with MOB_WIP_BRANCH_PREFIX
-	DoneSquash                     string // override with MOB_DONE_SQUASH
-	Timer                          string // override with MOB_TIMER
-	TimerRoom                      string // override with MOB_TIMER_ROOM
-	TimerLocal                     bool   // override with MOB_TIMER_LOCAL
-	TimerRoomUseWipBranchQualifier bool   // override with MOB_TIMER_ROOM_USE_WIP_BRANCH_QUALIFIER
-	TimerUser                      string // override with MOB_TIMER_USER
-	TimerUrl                       string // override with MOB_TIMER_URL
+	CliName                        string       // override with MOB_CLI_NAME
+	RemoteName                     string       // override with MOB_REMOTE_NAME
+	WipCommitMessage               string       // override with MOB_WIP_COMMIT_MESSAGE
+	GitHooksEnabled                bool         // override with MOB_GIT_HOOKS_ENABLED
+	RequireCommitMessage           bool         // override with MOB_REQUIRE_COMMIT_MESSAGE
+	VoiceCommand                   string       // override with MOB_VOICE_COMMAND
+	VoiceMessage                         string // override with MOB_VOICE_MESSAGE
+	NotifyCommand                        string // override with MOB_NOTIFY_COMMAND
+	NotifyMessage                        string // override with MOB_NOTIFY_MESSAGE
+	NextStay                             bool   // override with MOB_NEXT_STAY
+	StartIncludeUncommittedChanges       bool   // override with MOB_START_INCLUDE_UNCOMMITTED_CHANGES variable
+	StashName                            string // override with MOB_STASH_NAME
+	WipBranchQualifier                   string // override with MOB_WIP_BRANCH_QUALIFIER
+	WipBranchQualifierSeparator          string // override with MOB_WIP_BRANCH_QUALIFIER_SEPARATOR
+	WipBranchPrefix                      string // override with MOB_WIP_BRANCH_PREFIX
+	DoneSquash                           string // override with MOB_DONE_SQUASH
+	WriteLastModifiedFileInCommitMessage bool   // override with MOB_WRITE_LAST_MODIFIED_FILE_IN_COMMIT_MESSAGE
```

Comment:
Why do we need a flag for this? Why not always do this, if possible?

Reply:
I'm not sure if we would get any problems, if one has the new version of mob with this feature and one has an old version. If the one with the old version does a mod done, it maybe could produce problems while squashing, because his mob version would not recognize the commit as a WIP commit.

Reply:
Hm, I would take that into account.

Reply:
I thought about this the last night. We should be really careful. If we would break the squashing feature of older version. Maybe we should mark this feature as experimental first and let people activate it, but then adjust the squashing. After a few releases we could remove the option to activate it and activate it by default. Then we would not just break the squashing if one of the team uses one version lower then the others. What do you think about that?

Reply:
I think not a lot of users use the squashing feature. That’s why I don’t think this migration path is necessary. We can increase the major version to signal the major change.

Reply:
ok